g2p/transducer/__init__.py

```diff
@@ -264,7 +266,10 @@ def find_monotonic_segments(alignments):
                 else:
                     assert oedge[0] is not None
                     iend = max(iend, oedge[0])
-                    if iedge[1] is not None:
+                    # Replace None with not-None
+                    if oend is None:
+                        oend = iedge[1]
```

Comment:
I don't fully understand the logic, and this specific case is not covered by unit testing (maybe add a test to cover it?) but is it using the right non-None thing? Are you mapping to the nearest thing you can?

Reply:
Oh - I should make sure this gets covered.  We have to always ensure that no `None` escapes into the computed alignments - since a `None` means a deletion, it would be replaced with the next available index.  In practice this may not happen but I think I can still provoke it in a test case, so I will do that.

Reply:
this got removed because we now ensure `None`  doesn't make it there upstream from here